tests/test_handle_files.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import tempfile
+import shutil
+from os.path import abspath, dirname, join
+
+from kodi_addon_checker.handle_files import find_file
+from kodi_addon_checker.handle_files import create_file_index
+from kodi_addon_checker.handle_files import find_file_recursive as FFR
+
+HERE = abspath(dirname(__file__))
+
+
+class TestFindFile(unittest.TestCase):
+
+    def setUp(self):
+        self.path = tempfile.mkdtemp()
+        self.file = join(self.path, "addon.txt")
+
+    def test_find_file_is_none(self):
+        self.assertIsNone(find_file("randomname", self.path))
+
+    def test_find_file_is_not_none(self):
+        with open(self.file, "w+"):
+            self.assertIsNotNone(find_file("addon.txt", self.path))
+
+    def teardown(self):
+        shutil.rmtree(self.path)
+
+
+class TestFindFileRecursive(unittest.TestCase):
+
+    def setUp(self):
+        self.path = tempfile.mkdtemp()
+        self.file = join(self.path, "addon.txt")
+
+    def test_find_file_is_none(self):
+        self.assertIsNone(FFR("randomname", self.path))
+
+    def test_find_file_is_not_none(self):
```

Comment:
This test is fine, but I would like to also have a test for a file being deeper in a directory structure and still being correctly found by this.